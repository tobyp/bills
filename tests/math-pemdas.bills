#!./bills
A	B.	(2+3)*5
C	D.	2+3*5
E	F.	-3+4
G	H.	-(3+4)
I	J.	3-5-7
