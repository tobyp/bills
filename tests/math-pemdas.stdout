todo: B pays 25.0 to A
todo: D pays 17.0 to C
todo: F pays 1.0 to E
todo: G pays 7.0 to H
todo: I pays 9.0 to J
